SpaceToStudy/ui/pages/explore_offers/explore_offers_page.py

```diff
@@ -103,9 +108,18 @@ def click_go_to_categories_btn(self):
     def click_back_to_all_subject(self):
         return self.get_back_to_all_subject().click()
 
-    def get_list_of_filtered_offers(self) -> list:
+    def get_list_of_offers_grid_card(self) -> list:
+        offers = self.driver.find_element(*LIST_OF_OFFERS).find_elements(*OFFER_ITEM)
+        self._list_of_offers_grid_card = []
+        if self.driver.find_element(*GRID_CARD).is_displayed():
+            for offer in offers:
+                self._list_of_offers_grid_card.append(GridCardComponent(offer))
```

Comment:
offers = self.driver.find_elements(*GRID_CARD)
self._list_of_offers_grid_card = [GridCardComponent(offer) for offer in offers]